main.js

```diff
@@ -0,0 +1,7 @@
+let question = prompt( '日本の首都は?' );
+
+if (question = prompt( '日本の首都は?' ) === '東京') {
```

Comment:
１行目と３行目のpromptの定義が重複しており、質問が２回出てきますね。
（おそらくループさせようとして試したもの...？？）

`let question = prompt( '日本の首都は?' );`
の定義で、回答がquestion変数に格納されますので、if文は以下のように出来ます！
```suggestion
if (question === '東京') {
```

また、そうするとquestionは`const`で定義出来ます。
```
const question = prompt( '日本の首都は?' );
```
letは再代入される場合、constは再代入されない場合に使用します。
できるだけletは使用しないコードが書けると良いので、意識してみてください！